README.md

```diff
@@ -49,17 +53,43 @@ This action can be configured two different ways -- either in combination with [
     PROGRAMID: ${{ secrets.CM_PROGRAM_ID }}
 ```
 
+#### Using OAUTH integration
+
+* `CLIENTID` - Client ID for the project in the I/O Developer Console
+* `CLIENTSECRET` - Client Secret for the project in the I/O Developer Console
+* `TECHNICALACCOUNTID` - Technical Account ID for the project in the I/O Developer Console
+* `TECHNICALACCOUNTEMAIL` - Technical Account Email for the project in the I/O Developer Console
+* `IMSORGID` - IMS Organization ID for the project in the I/O Developer Console
+* `PROGRAMID` - Cloud Manager Program ID
+* `PIPELINEID` - Cloud Manager Pipeline ID
+
+```
+- name: Create Execution
+  uses: adobe/aio-cloudmanager-create-execution-action@v1.0.5
+  with:
+    CLIENTID: ${{ secrets.CM_CLIENT_ID }}
+    CLIENTSECRET: ${{ secrets.CM_CLIENT_SECRET }}
+    TECHNICALACCOUNTID: ${{ secrets.CM_TA_ID }}
+    TECHNICALACCOUNTEMAIL: ${{ secrets.CM_TA_EMAIL }}
+    IMSORGID: ${{ secrets.CM_ORG_ID }}
+    PIPELINEID: ${{ secrets.CM_PIPELINE_ID }}
+    PROGRAMID: ${{ secrets.CM_PROGRAM_ID }}
+    OAUTHENABLED: true
+```
+
 ### Usage with aio-apps-action
 
 1. Add a step which uses `adobe/aio-apps-action` providing these required inputs:
 
 * `command` - set to `auth`
 * `CLIENTID` - Client ID for the project in the I/O Developer Console
 * `CLIENTSECRET` - Client Secret for the project in the I/O Developer Console
-* `TECHNICALACCOUNTID` - Technical Account Email for the project in the I/O Developer Console
+* `TECHNICALACCOUNTID` - Technical Account ID for the project in the I/O Developer Console
+* `TECHNICALACCOUNTEMAIL` - Technical Account Email for the project in the I/O Developer Console (Only for OAuth integration)
 * `IMSORGID` - IMS Organization ID for the project in the I/O Developer Console
-* `SCOPES` - set to `ent_cloudmgr_sdk` **at minimum**
-* `KEY` - Private key for the project in the I/O Developer Console
+* `SCOPES` - set to `ent_cloudmgr_sdk` **at minimum** for JWT and `openid, AdobeID, read_organizations, additional_info.projectedProductContext, read_pc.dma_aem_ams` for OAuth
+* `KEY` - Private key for the project in the I/O Developer Console (Only for JWT integration)
+* `OAUTHENABLED` - set to true for OAuth-type integrations (required for OAuth integration; optional for JWT integration)
```

Comment:
For JWT integration, shouldn't this param `OAUTHENABLED` be false?

Reply:
if you check the `action.yml`, it is set to false by default